.travis.yml

```diff
@@ -46,8 +46,12 @@ before_script:
 # -> remove legacy hdf5 and mpi packages/deps from travis
   - sudo apt-get remove -qq libhdf5*
   - sudo apt-get remove -qq libopenmpi* openmpi-bin || { echo "OpenMPI not installed"; }
+  - sudo apt-get clean
+  - sudo sh -c 'echo "Acquire::http::Pipeline-Depth 0;\nAcquire::http::No-Cache true;\nAcquire::BrokenProxy    true;" > /etc/apt/apt.conf.d/99fixbadproxy'
```

Comment:
tried [this SO answer](https://askubuntu.com/questions/679233/failed-to-fetch-hash-sum-mismatch-tried-rm-apt-list-but-didnt-work) here...